src/HttpHeader.h

```diff
@@ -59,6 +59,8 @@ class HttpHeaderEntry
     void packInto(Packable *p) const;
     int getInt() const;
     int64_t getInt64() const;
+    /// packed length, including ": " and CRLF
+    size_t length() const { return name.length() + 2 + value.size() + 2; }
```

Comment:
I think we should be more explicit here because many different "lengths" may be associated with HttpHeaderEntry. For example:

```suggestion

    /// expected number of bytes written by packInto(), including ": " and CRLF
    size_t length() const { return name.length() + 2 + value.size() + 2; }
```

Please also add that empty line before the comment.